app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command_, file_to_move, path_to_move = command.split()
+        if command_ == "mv" and file_to_move != path_to_move.split("/")[-1]:
+            place_to_move = path_to_move.split("/")[:-1]
```

Comment:
Using `os.path.split()` is preferred over string manipulation to extract the filename from a path